### PR TITLE
Improvements to UI in web logged-out views

### PR DESCRIPTION
**Pull request by pfrazee**
Small UI update to logged out views so that we don't look quite so amateur

CleanShot 2023-08-30 at 16 56 45@2x
CleanShot 2023-08-30 at 16 56 54@2x

### CODE REVIEW

**Comment by ansh** [comment]
No option to skip onboarding on desktop web

**Comment by pfrazee** [comment]
yeah I dunno, right now it's 1 step so I'm kind of inclined to just get that in later

**Comment by ansh** [comment]
It'd be v useful to get the metrics to see how many people still skip it @pfrazee

**Review by ansh — APPROVED** [review]
Works well on all platforms and light/dark mode too

File: src/state/models/ui/create-account.ts [hunk 1]
```diff
@@ -125,7 +120,7 @@ export class CreateAccountModel {
       })
       track('Create Account')
     } catch (e: any) {
-      sessionReadySub.remove()
+      this.rootStore.onboarding.skip() // whoops jk
```
Comment (ansh) [inline src/state/models/ui/create-account.ts:123]: is this supposed to be here?

Comment (pfrazee) [reply src/state/models/ui/create-account.ts:123]: hah yes, I was being a bit goofy with my comment. I'll fix that

**Comment by pfrazee** [comment]
> It'd be v useful to get the metrics to see how many people still skip it @pfrazee

okay let's plan to get that in with the additional screens you've got planned

**Pull request merged**